### ISSUE TITLE
Home: Images must have an alternative text 

### ISSUE THREAD

**Issue by mmiron-HC** [mmiron-HC/WST-RSAMS#7]
URL: https://hcdev.openplus.ca/en
User Agent: Chrome/Firefox
language: both

Issue description: The images for each link seem to have a meaning to the visual user that it's a recall/alert for a food, health etc type post. These seemed to be marked as decoration as screen reader is not announcing them.
![image](https://user-images.githubusercontent.com/65728700/126989784-7fedda56-53aa-491b-a712-da10ff1abf83.png)

Steps to reproduce: Screen reader (NVDA)

Expected result: All of these images should have an alt text so the user knows what category it is.

**Comment by davidgotlieb** [comment]
﻿Since these are related to the category tag which is displayed below the alert/recall these icons are decorative.

**Comment by ndolter** [comment]
This ticket can be closed since the class = svg-inline and images are decorative.
